packages/sortable/src/components/SortableContext.tsx

```diff
@@ -85,30 +101,27 @@ export function SortableContext({
     previousItemsRef.current = items;
   }, [items]);
 
+  const globalActiveRef = useGlobalActiveRef(active?.id || null);
   const contextValue = useMemo(
     (): ContextDescriptor => ({
-      activeIndex,
       containerId,
       disabled,
-      disableTransforms,
+      useShouldUseDragTransform: sortingAPI.useShouldUseDragTransform,
       items,
-      overIndex,
       useDragOverlay,
-      sortedRects: getSortedRects(items, droppableRects),
-      strategy,
+      useMyNewIndex: sortingAPI.useMyNewIndex,
+      globalActiveRef,
+      useMyStrategyValue: sortingAPI.useMyStrategyValue,
     }),
     // eslint-disable-next-line react-hooks/exhaustive-deps
     [
-      activeIndex,
       containerId,
       disabled.draggable,
       disabled.droppable,
-      disableTransforms,
-      items,
-      overIndex,
-      droppableRects,
+      // items,
```

Comment:
@alissaVrk I noticed significant performance increases in my latest tests! However, it seems that with 'items' absent from the dependency array (SortableContext > contextValue), the context provider items value is not updated in conditional rendering situations, preventing sorting functionality.

Reply:
I think it was a mistake...
forgot to uncomment. - in theory, it would be nice to change most of the stories to be optimized for performance, so the tests will catch these things 

by the way, this is one of the dependencies that mean that all items are rendered on drop

Reply:
so the described issue does not exists anymore?

Reply:
The PR isn't merged..

Reply:
@hexwit seems like it